### ISSUE TITLE
Spotify Music Recommender

### ISSUE THREAD

**Issue by prathimacode-hub** [prathimacode-hub/ML-ProjectKart#356]
Hey LGM-SoC participants, good to see you.

This project will help us in providing a good recommendation from spotify playlist.

Dataset : https://www.kaggle.com/asifsadmine/spotify-playlists-dataset

If you like to work on this issue, follow the given guidelines for code as well as for the README.

**Comment by swetha28** [comment]
Hi. I would like to work on this issue. Can you please assign it to me?

**Comment by palak740** [comment]
Hi, @prathimacode-hub , I am interested in taking this project for contributing. Kindly assign it to me under DevIncept21 30-day contributing program.

**Comment by prathimacode-hub** [comment]
Are you interesting to work on this issue. Kindly elaborate your approach on how would you proceed with this issue and also the task you're taking up? Model Creation, Front End or Back End. @swetha28

**Comment by swetha28** [comment]
> Are you interesting to work on this issue. Kindly elaborate your approach on how would you proceed with this issue and also the task you're taking up? Model Creation, Front End or Back End. @swetha28

I will do the model creation part. First, I will analyze the dataset using python and build an appropriate ML algorithm.

**Comment by prathimacode-hub** [comment]
You have to use multiple algorithms and compare the accuracy. It will improve efficiency. It should be brief enough. If you're ok with it. I shall assign the issue. @swetha28

**Comment by swetha28** [comment]
@prathimacode-hub Yeah. I will develop multiple algorithms and then compare the accuracy.

**Comment by prathimacode-hub** [comment]
Issue assigned. @swetha28

**Comment by prathimacode-hub** [comment]
Its been 23days and no update from you yet neither PR turned up. @swetha28

**Comment by swetha28** [comment]
As I mentioned in the channel, this dataset is very huge. Shall I take any other Spotify dataset from Kaggle?

**Comment by prathimacode-hub** [comment]
Yes, you can take up another dataset to work on from kaggle. @swetha28

**Comment by prathimacode-hub** [comment]
Update please. @swetha28

**Comment by prathimacode-hub** [comment]
9th is the last date to accept PR's. If you wouldn't do it by then, obviously you can submit later but points won't be counted towards DCP program. @swetha28

**Comment by prathimacode-hub** [comment]
Sorry, label has to removed now as pe their instructions. You can work on it if possible in 2 days or so ASAP. But it won't be counted under the program as time is up for it. @swetha28

**Comment by prathimacode-hub** [comment]
Any update? @swetha28
